Cargo.lock

```diff
@@ -6957,7 +6957,7 @@ version = "1.6.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "97fee6b57c6a41524a810daee9286c02d7752c4253064d0b05472833a438f675"
 dependencies = [
- "cfg-if 1.0.0",
```

Comment:
Is this intended?

Reply:
i think it is a transitive dependency as part of updating h2.